apps/web/src/containers/TransactionScreen/components/info-box.tsx

```diff
@@ -0,0 +1,111 @@
+import { Box, styled, Typography, Tabs, Tab } from '@mui/material';
+import { useState, ReactNode } from 'react';
+
+// Types for the component props
+export interface TabItem {
+  label: string;
+  content: ReactNode;
+}
+
+interface InfoBoxProps {
+  tabs: TabItem[];
+}
+
+// Info Box Component
+const InfoBox = ({ tabs }: InfoBoxProps) => {
+  const [selectedTab, setSelectedTab] = useState(0);
+
+  const handleTabChange = (_event: React.SyntheticEvent, newValue: number) => {
+    setSelectedTab(newValue);
+  };
+
+  return (
+    <InfoBoxContainer>
+      <TabsContainer>
+        <CustomTabs
+          value={selectedTab}
+          onChange={handleTabChange}
+          variant="fullWidth"
+        >
+          {tabs.map((tab, index) => (
+            <CustomTab key={index} label={tab.label} />
+          ))}
+        </CustomTabs>
+      </TabsContainer>
+
+      <ContentContainer>{tabs[selectedTab]?.content}</ContentContainer>
```

Comment:
I'll need some reusable components for the content container here. See the markup here:
https://docs.google.com/document/d/1Y90uuxqnYGjJQXsLTj0peQO-hO9JTiDaEkRdQzNHSKo/edit?tab=t.0#heading=h.exef1jgp70vr

I want to be able to specify an array with a shape like:
`{ sectionTitle: string, items: { label: ReactNode,  value: ReactNode }[] }[]`

And have it populate this section with a proper <hr> component between sections